{"level":"error","message":"Uncaught internal server error. { Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:953:11)\n    at exports._exceptionWithHostPort (util.js:976:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:953:11)\n    at exports._exceptionWithHostPort (util.js:976:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)","timestamp":"2016-05-18T04:19:02.400Z"}
{"level":"error","message":"Uncaught internal server error. { Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:953:11)\n    at exports._exceptionWithHostPort (util.js:976:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:953:11)\n    at exports._exceptionWithHostPort (util.js:976:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)","timestamp":"2016-05-18T04:19:04.392Z"}
{"level":"error","message":"Uncaught internal server error. { Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:953:11)\n    at exports._exceptionWithHostPort (util.js:976:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)\n  name: 'MongoError',\n  message: 'connect ECONNREFUSED 127.0.0.1:27017' } Error: connect ECONNREFUSED 127.0.0.1:27017\n    at Object.exports._errnoException (util.js:953:11)\n    at exports._exceptionWithHostPort (util.js:976:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)","timestamp":"2016-05-18T04:19:12.139Z"}
